hardware-api.md

```diff
@@ -128,6 +128,16 @@ i2c.transfer(new Buffer([0xde, 0xad, 0xbe, 0xef]), function (err, rx) {
 })
 ```
 
+### Terms Used
+
+The following section uses non-inclusive, technical terms provided by an industry standard. They are used here in an explicitly technical manner and only to be in accurate in describing the following wire communication protocols. The following terms are presented as defined here:
```

Comment:
I had some typos and incorrectly structured language in my original text...

(this includes @wyze's comment)

| Present | Replace With... |
| ---------- | -------------------- |
| The following section uses non-inclusive, technical terms provided by an industry standard. | The following sections use industry standard technical terms that are considered non-inclusive. |
| be in accurate | be accurate | 
| the following wire communication protocols | these wire communication protocols | 
| The following terms are presented as defined here: | The terms are used strictly as defined here: |